packages/blocks/src/api/registration.js

```diff
@@ -177,7 +176,10 @@ export function unstable__bootstrapServerSideBlockDefinitions( definitions ) {
 			continue;
 		}
 		serverSideBlockDefinitions[ blockName ] = mapKeys(
-			pickBy( definitions[ blockName ], ( value ) => ! isNil( value ) ),
+			pickBy(
+				definitions[ blockName ],
+				( value ) => value !== null && typeof value !== 'undefined'
```

Comment:
Why checking `typeof` and not value here?
```suggestion
				( value ) => value !== null && value !== undefined
```

Reply:
I usually prefer relying on `typeof value === 'undefined'` because it will never throw and is generally safer. 
`value === undefined` on the other hand will also throw a `ReferenceError` when performing the check. 

Is there a good reason to choose the less safe approach?

Reply:
I think if we were worried about a `ReferenceError` we should handle it accordingly in a piece of code, and here we would have the error in the first check with `null`, wouldn't we?

So here since `undefined` is a primitive, we can just do that.

Reply:
Fair enough 👍  Updated in d62b5ab49715b07e069adddc95818adf6ad88e70

Reply:
Thanks! I know it was a nitpick, I would have approved either way. 😄